### ISSUE TITLE
Tabs: focus outline partially obscured for active tab

### ISSUE THREAD

**Issue by daneah** [ithaka/pharos#473]
**Expected behavior**

When a tab is focused, its full focus outline is visible

**Actual behavior**

The focus outline along the top of all tabs, and the outside edge for first/last tabs, is not visible

**Steps to reproduce the issue**

1. Visit a page with a tab
2. Click on a tab

**Screenshots or code**

<img width="245" alt="Screen Shot 2023-01-17 at 14 41 56" src="https://user-images.githubusercontent.com/1808306/212996353-f0adbb7d-49ab-4609-a654-824c8c521e3b.png">

**Pharos version**

12.16.0+ (introduced in #457)

**Your environment**

- OS: macOS
- Browser: All tried

**Additional information**

We can likely fix this by adjusting the `padding` on the `.tab__list` element.
